target/consensource-compose
# Rename client -> UI

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

This PR refactors instances of the phrase `client` with `ui`. This is done to match the updated name of the repo - `ConsenSource UI`.

Also gives executable permission to the `docker-helper.sh` script.

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

1.  `./docker-helper.sh --build ui` 
2. `./docker-helper.sh --run ui` and verify it succeeds by visiting `http://localhost:8080/`
